src/main/java/net/javadiscord/javabot/systems/moderation/report/ReportCommand.java

```diff
@@ -0,0 +1,38 @@
+package net.javadiscord.javabot.systems.moderation.report;
+
+import lombok.extern.slf4j.Slf4j;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.User;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.dv8tion.jda.api.requests.restaction.WebhookMessageAction;
+import net.javadiscord.javabot.systems.moderation.ModerateUserCommand;
+
+import javax.annotation.Nonnull;
+
+/**
+ * <h3>This class represents the /report command.</h3>
+ */
+@Slf4j
+public class ReportCommand extends ModerateUserCommand {
```

Comment:
`ModerateUserCommand` does not allow performing actions on other users with higher roles (`canInteract` check). This makes it impossible to report anyone with a role (e.g. question-ping) if the reporting user doesn't have any roles (or only lower roles).

One solution to this problem would be binding the `canInteract` check in `ModerateCommand` to `requireStaff`.